### DISCUSSION TITLE
Easy Online Buildtool

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#16015] [Q&A] [answered]
I'm thinking of a click and compile webbased buildtool.

The idea is that you can select options by checkboxed and/or dropdown boxes. After pressing build, the image is created and after a while ready for download.

Is there allready something for this or would it be a nice to have?

I think I can make this tool myself.. Would be a nice (little) project I think..

**Comment by arendst** [comment, marked as answer]
Search for tasmocompiler.

**Reply by jeroenst** [reply]
Thanx, i will host that asap.

**Comment by barbudor** [comment]
It's already hosted on github

**Reply by jeroenst** [reply]
Thanx, i'm now also hosting it myself on docker for easy access
